src/WebJobs.Script.Grpc/Abstractions/WorkerDescription.cs

```diff
@@ -15,6 +15,12 @@ public class WorkerDescription
         [JsonProperty(PropertyName = "language")]
         public string Language { get; set; }
 
+        /// <summary>
+        /// Gets or sets the default language version.
+        /// </summary>
+        [JsonProperty(PropertyName = "defaultLanguageVersion")]
```

Comment:
Add default OS as well so that worker can opt in for a default OS

Reply:
As noted below, would need more elaboration on the purpose of this given that we are already determining the OS in the Host and will also validate if it is supported.